src/spawn_players.c

```diff
@@ -465,7 +465,7 @@ void func_80039DA4(void) {
         4, 5, 6, 7,
     };
 
-    if (((gCupCourseSelection == CUP_COURSE_ONE) && (D_8016556E == 0)) ||
+    if (((gCupCourseSelection == MUSHROOM_CUP) && (D_8016556E == 0)) ||
```

Comment:
Could you verify this that this is supposed to be the cup name rather than the course in the cup?